src/style.css

```
@@ -1 +1,28 @@
-/* styles go here */
+label {
+  display: block;
+  margin-bottom: 10px;
+}
+
+.age input {
+  width: 65px;
+}
+
+.fieldset {
+  margin-top: 20px;
+}
+
+.what_1 {
+  margin-top: 10px;
+}
```

you can use comma
```suggestion
.fieldset,
.what_1 {
  margin-top: 20px;
}
```

or just add some common classes